compiler-rt/lib/interception/interception_win.cpp

```diff
@@ -634,7 +656,6 @@ static size_t GetInstructionSize(uptr address, size_t* rel_offset = nullptr) {
     case 0x018a:  // mov al, byte ptr [rcx]
       return 2;
 
-    case 0x058A:  // 8A 05 XX XX XX XX : mov al, byte ptr [XX XX XX XX]
```

Comment:
Just want to note that this line is being moved to below (line 668), which fixes a wrong change from https://github.com/llvm/llvm-project/commit/3bd8f4e0a05f2eecc2029ff05d21f19e74b6eb67#diff-ca7181301de35b7cb405b0809770e12d68a5466b396c7f0e5cf61ab2145c828dR637-R649 .

It may be worth making a separate review for this single change (or even just committing the fix directly) to have it fixed quickly.

Reply:
Splitted of this PR: GetInstructionSize: Fix `8A 05 ...` to return 6 again.
https://github.com/llvm/llvm-project/pull/116889
Thank you, hope it is ok when I resolve this paragraph.